dist/index.js

```diff
@@ -7612,9 +7612,13 @@ async function get(url0, pageIdxs) {
       const headers = {
         'user-agent': 'setup-beam',
       }
-      if (process.env.GITHUB_TOKEN) {
-        headers.authorization = `Bearer ${process.env.GITHUB_TOKEN}`
+      const GithubToken =
+        getInput('github-token', false) || process.env.GITHUB_TOKEN
```

Comment:
Why do we need the `||`. It's either defined by the consumer or default, right? Unless you+re thinking the consumer might put a falsy value there.

Reply:
Honestly I'm not sure we need the `or` at all. It depends on whether we want to support providing a token that way. For a second I thought we needed it for backwards compat with the previous release, but since `github.token` is functionally the same, it shouldn't matter (i.e., there will be no breaking change).  Thoughts?

Reply:
I'm Ok with either removing it (not sure it's required) or keeping it (doesn't hurt, but it may be confusing). Your choice.